obal/data/playbooks/nightly.yml

```diff
@@ -0,0 +1,8 @@
+---
+- hosts: packages
+  serial: 1
+  gather_facts: no
+  roles:
```

Comment:
Should this be using the nightly role?

Reply:
I guess this might be my copy-paste error and why it didn't work.